giveaways/models.py

```diff
@@ -349,7 +349,7 @@ def get_next_edit_time(self):
                 time.time() + 60
             )  # middle case, its not greater than 5 minutes but greater than a minute.
 
-    async def hdm(self, host, jump_url, prize, winners):
+    async def hdm(self, host, jump_url, prize, winners, guild):
```

Comment:
Don't need an extra guild argument here, you already have `self.guild` available which can be used since each giveaway object is tied to its own guild